### PR TITLE
Load point clouds from CDF

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: viewer/webpack.config.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ module.exports = env => {
       globalObject: `(typeof self !== 'undefined' ? self : this)`,
       libraryTarget: 'umd',
     },
-    devtool: development ? "inline-cheap-module-source-map" : "source-map",
+    devtool: development ? "inline-source-map" : "source-map",
```
Comment (larsmoa) [inline viewer/webpack.config.js]: This doesn't really belong here, but was added because source maps didn't work in chrome.

File: examples/src/simple-pointcloud.ts [hunk 2]
```diff
@@ -3,31 +3,30 @@
...
+import { CogniteClient } from '@cognite/sdk';
+import { toThreeJsBox3 } from '@cognite/reveal/threejs';
 
 CameraControls.install({ THREE });
 
 async function main() {
-  const modelUrl = new URL(location.href).searchParams.get('model') || '/transformer-point-cloud/cloud.js';
+  const urlParams = new URL(location.href).searchParams;
+  const modelIdentifier = urlParams.get('model') || '/transformer-point-cloud/cloud.js';
+  const project = urlParams.get('project') || '3ddemo';
```
Comment (dragly) [inline examples/src/simple-pointcloud.ts]: Do not leak internal tenant names. Only we have access to this tenant.

File: examples/src/simple-pointcloud.ts [hunk 3]
```diff
@@ -112,4 +102,16 @@ function initializeGui(node: reveal.internal.PotreeNodeWrapper, handleSettingsCh
     });
 }
 
+async function createPointCloudModel(model: string, project: string): Promise<reveal.PointCloudModel> {
```
Comment (dragly) [inline examples/src/simple-pointcloud.ts]: Add a verification here that project is not specified if the model is local.

File: viewer/src/datasources/cognitesdk/pointcloud/createPointCloudModel.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    const transform: SectorModelTransformation = {
+      modelMatrix: identity,
+      inverseModelMatrix: identity
+    };
+    return [await loaderPromise, transform];
+  };
+  return [fetchPointCloud];
+}
+
+function initializeXHRRequestHeaders(sdk: CogniteClient) {
```
Comment (dragly) [inline viewer/src/datasources/cognitesdk/pointcloud/createPointCloudModel.ts]: Nitpick: `initializeXhrRequestHeaders`

(we have not really discussed naming convention for abbreviations, but I prefer camel case all the way even for abbreviations)

File: viewer/src/models/cad/initializeSectorLoader.ts [hunk 5]
```diff
@@ -61,15 +61,15 @@ export function initializeSectorLoader<T>(
...
-    if (needsRedraw) {
-      console.log(
-        `activateSectors() [wanted: ${wantedSectorIds.size} ` +
-          `new: ${newSectorIds.size}` +
-          ` discarded: ${discardedSectorIds.size}` +
-          ` active: ${activeSectorIds.size}` +
-          ` in-flight: ${activeSectorRequests.size}] time=${(performance.now() - start).toPrecision(2)} ms`
-      );
-    }
+    // if (needsRedraw) {
```
Comment (dragly) [inline viewer/src/models/cad/initializeSectorLoader.ts]: Unrelated change?

File: viewer/src/utils/CogniteClient3dV2Extensions.ts [hunk 6]
```diff
@@ -0,0 +1,115 @@
+/*!
```
Comment (dragly) [inline viewer/src/utils/CogniteClient3dV2Extensions.ts]: Perhaps scope the file to Model3D in the name as well, in case we need more over time?

`CogniteClientModel3DV2Extensions`

Perhaps also drop the V2 in the name? I suppose extensions can just be non-versioned and work with whichever API version is necessary.

File: viewer/src/views/threejs/pointcloud/PotreeGroupWrapper.ts [hunk 7]
```diff
@@ -31,7 +31,9 @@ export class PotreeGroupWrapper extends THREE.Object3D {
     this.name = 'Potree point cloud wrapper';
     this.add(this.potreeGroup);
 
-    this.onAfterRender = this.resetNeedsRedraw;
+    const onAfterRenderTrigger = new THREE.Mesh(new THREE.Geometry());
+    onAfterRenderTrigger.onAfterRender = () => this.resetNeedsRedraw();
```
Comment (dragly) [inline viewer/src/views/threejs/pointcloud/PotreeGroupWrapper.ts]: Great `this` goodness :tada:

File: examples/src/simple-pointcloud.ts (hunk 2 above)
Comment (larsmoa) [reply examples/src/simple-pointcloud.ts]: Good point :)

File: viewer/src/models/cad/initializeSectorLoader.ts (hunk 5 above)
Comment (larsmoa) [reply viewer/src/models/cad/initializeSectorLoader.ts]: I guess, but I got complaints from tslint.

File: viewer/src/utils/CogniteClient3dV2Extensions.ts (hunk 6 above)
Comment (larsmoa) [reply viewer/src/utils/CogniteClient3dV2Extensions.ts]: I agree with dropping V2, but I think 3d is sufficient - why is adding Model necessary?
